Formula/d/dolt.rb

```diff
@@ -27,6 +27,27 @@ def install
     end
   end
 
+  def logdir
+    var/"log"
+  end
+
+  def datadir
+    var/"dolt"
+  end
+
+  def post_install
+    logdir.mkpath
+    datadir.mkpath
+  end
+
+  service do
+    run [opt_bin/"dolt", "sql-server"]
+    keep_alive true
+    log_path f.logdir/"dolt.log"
+    error_log_path f.logdir/"dolt.error.log"
+    working_dir f.datadir
+  end
```

Comment:
No need to make those methods on the function
```suggestion
  def post_install
    (var/"log").mkpath unless (var/"log").exist?
    (var/"dolt").mkpath
  end

  service do
    run [opt_bin/"dolt", "sql-server"]
    keep_alive true
    log_path var/"log/dolt.log"
    error_log_path var/"log/dolt.error.log"
    working_dir var/"dolt"
  end
```